### PR TITLE
Expense Creation: DE Flow

### PR DESCRIPTION
**Pull request by shreyanshss7**

### CODE REVIEW

File: fyle_slack_app/slack/events/tasks.py [hunk 1]
```diff
@@ -141,6 +144,24 @@ def handle_file_shared(file_id: str, user_id: str, team_id: str):
...
+                "file_name": file_info['file']['name'],
+                "file_content": encoded_file,
+                "source": "SLACK"
+            }
+        }
+        try:
+            expense = fyle_utils.extract_expense_from_receipt(receipt_payload, user.fyle_refresh_token)
+            view_expense_message = expense_messages.view_expense_message(expense, user)
+            slack_client.chat_postMessage(channel=user.slack_dm_channel_id, blocks=view_expense_message, thread_ts=message_ts, reply_broadcast=True)
+        except assertions.InvalidUsage:
```
Comment (gokulk16) [inline fyle_slack_app/slack/events/tasks.py:162]: there seems to be other possible error codes from API. please handle those as well
https://fyle.stoplight.io/docs/fyle-platform-docs/555e474294850-create-expense-from-a-receipt

File: fyle_slack_app/slack/events/tasks.py [hunk 2]
```diff
@@ -141,6 +144,24 @@ def handle_file_shared(file_id: str, user_id: str, team_id: str):
     # This else block means file has been shared as a new message and an expense will be created with the file as receipt
     # i.e. data extraction flow
     else:
+        expense_creation_message = ':hourglass_flowing_sand: Uploading receipt and creating an expense :zap:'
```
Comment (gokulk16) [inline fyle_slack_app/slack/events/tasks.py:147]: :hourglass_flowing_sand: creating an expense and uploading receipt :zap:

**Review by gokulk16 — CHANGES_REQUESTED** [review]

File: fyle_slack_app/slack/events/tasks.py [hunk 3]
```diff
@@ -141,6 +144,24 @@ def handle_file_shared(file_id: str, user_id: str, team_id: str):
     # This else block means file has been shared as a new message and an expense will be created with the file as receipt
     # i.e. data extraction flow
     else:
+        expense_creation_message = ':hourglass_flowing_sand: creating an expense and uploading receipt :zap:'
```
Comment (jatzz10) [inline fyle_slack_app/slack/events/tasks.py:147]: `.... Creating an expense .....`
Capital 'C'

**Review by jatzz10 — APPROVED** [review]
@shreyanshs7 Added one comment - please fix
Rest lgtm

**Pull request merged**